returnn/datasets/meta.py

```diff
@@ -180,6 +180,7 @@ def __init__(self,
                seq_list_file=None,
                seq_order_control_dataset=None,
                seq_lens_file=None,
+               default_data_key="data",
```

Comment:
Why is the option here, but the attrib is in the base class? The option should be right where you have the attrib.